src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return 230;
+  }
+
+  if (days > 3 && days <= 6) {
+    return 220;
+  }
+
+  if (days >= 3) {
```

Comment:
The conditions on lines 11 and 15 overlap for the case when 'days' is exactly 3, which could introduce bugs. Since you have a return statement in each if block, you can simplify the conditions by checking for the discount cases in descending order and then returning the base cost for all other cases.